spec/database_helpers.rb

```diff
@@ -0,0 +1,7 @@
+require 'pg'
+
+def persisted_data(id:)
+  connection = PG.connect(dbname: 'chitter_test')
+  result = connection.query("SELECT * FROM peeps WHERE id = #{id};")
+  result.first
+end
```

Comment:
I didn't know if I needed the persisted data/id stuff. Since we don't have to add a feature for updating peeps, the only thing we'd use it for is testing that the peeps we've added to the database have the same id as the ones we return from the database. And we can test which peeps get returned against our expectations just by comparing the string. It doesn't hurt, but I don't know if it's a useful test in this situation.